src/engine/sync/synccontrol.cpp

```diff
@@ -278,13 +278,13 @@ void SyncControl::reinitLeaderParams(
         kLogger.trace() << "SyncControl::reinitLeaderParams" << getGroup()
                         << beatDistance << baseBpm << bpm;
     }
-    m_leaderBpmAdjustFactor = determineBpmMultiplier(fileBpm(), baseBpm);
+    m_leaderBpmAdjustFactor = determineBpmMultiplier(mixxx::Bpm(m_pBpm->get()), bpm);
     updateLeaderBpm(bpm);
     updateLeaderBeatDistance(beatDistance);
 }
 
 double SyncControl::determineBpmMultiplier(mixxx::Bpm myBpm, mixxx::Bpm targetBpm) const {
-    if (!myBpm.isValid() || !targetBpm.isValid()) {
+    if (!myBpm.isValid() || !targetBpm.isValid() || myBpm.isNull() || targetBpm.isNull()) {
```

Comment:
is null ever a valid bpm? seems like isValid could check for null value rather than having a separate function.

Reply:
As part of my routine, I sometime makes breaks where I slide the tempo fader all the way to zeo (with a 100% range), meaning the final bpm will be `0.00`, on potentially the leader deck. Technically, `0.00` is a valid BPM, thus why the `isNull` introduction

Reply:
isValidValue only returns true if `kValueMin < value;`, so 0.0 is not a valid bpm by that function, so the original check should be fine.

Reply:
Ah indeed. I thought I had seen the min value being included in the valid range. 
I removed the above